docs/baseband/tutorials/getting_started.rst

```diff
@@ -0,0 +1,317 @@
+again we would get the next 12 samples.  If we instead called  ``fh.read()``, it
+would read from the pointer's *current* position to the end of the file.  If we
+wanted all the data in one array, we would move the file pointer back to the
+start of file, using ``fh.seek``, before reading::
+
+    >>> fh.seek(0)      # Seek to sample 0.  Seek returns its offset in counts.
+    0
+    >>> d_complete = fh.read()
+    >>> d_complete.shape
+    (40000, 8)
+
+.. note::
+
+    ``fh.read()`` returns a **copy** of the data from ``fh``.
+
+We can also move the pointer with respect to the end of file by passing ``2``
+as a second argument (as with `io.BufferedReader` pointers)::
+
+    >>> fh.seek(-100, 2)    # Second arg is 0 (start of file) by default.
+    39900
+    >>> d_end = fh.read(100)
+    >>> np.array_equal(d_complete[-100:], d_end)
+    True
+
+``-100`` means 100 counts before the end of file, so ``d_end`` is equal to
+the last 100 entries of ``d_complete``.  Baseband only keeps the most recently
+accessed data frame in memory, so selective decoding using ``seek`` and
+``read`` is useful when examining extremely large files.
+
+To determine where the pointer is located, we use ``fh.tell()``::
+
+    >>> fh.tell()
+    40000
+    >>> fh.close()
+
+Seeking and Telling in Time With the Sample Pointer
+---------------------------------------------------
+
+We can use ``seek`` and ``tell`` with units of time rather than samples.  To do
+this with ``tell``, we can pass an appropriate `astropy.units.Unit` object to
+its optional ``unit`` parameter::
+
+    >>> fh = vdif.open(SAMPLE_VDIF, 'rs')
+    >>> fh.seek(40000)
+    40000
+    >>> fh.tell(unit=u.ms)
+    <Quantity 1.25 ms>
+
+Passing the string ``'time'`` reports the pointer's location in absolute time::
+
+    >>> fh.tell(unit='time')
+    <Time object: scale='utc' format='isot' value=2014-06-16T05:56:07.001250000>
+
+We can also pass an absolute `astropy.time.Time`, or a positive or negative time
+difference `~astropy.time.TimeDelta` or `astropy.units.Quantity` to ``seek``. 
+If the offset is a `~!astropy.time.Time` object, the second argument to seek is
+ignored.
+
+::
+
+    >>> from astropy.time.core import TimeDelta
+    >>> from astropy.time import Time
+    >>> fh.seek(TimeDelta(-5e-4, format='sec'), 2)  # Seek -0.5 ms from end.
+    24000
+    >>> fh.seek(0.25*u.ms, 1)  # Seek 0.25 ms from current position.
+    32000
+    >>> # Seek to time index 2014/06/16 5:56:07.001125
+    >>> fh.seek(Time('2014-06-16T05:56:07.001125'))
+    36000
+    >>> fh.close()
+
+Extracting Header Information
+-----------------------------
+
+The first header of the file is stored as the ``header0`` attribute of the
+stream reader object; it gives direct access to header properties via keyword
+lookup::
+
+    >>> with vdif.open(SAMPLE_VDIF, 'rs') as fh:
+    ...     header0 = fh.header0
+    >>> header0['frame_length']
+    629
+
+The full list of keywords is available by printing out ``header0.keys()``.
```

Comment:
Why not just show `header0`? It is not *that* long.

Reply:
This is a Python 2 compatibility problem.  In Python 3, printing out the keys returns an `odict_keys` object, while in Python 2 it returns a list of unicode strings.  We could just write it Python 3 style, then flag pytest to not check that example.  Would you know how to do that?

Reply:
Why not just do `>>> header0` and show what one gets?

Reply:
I just realized that printing `header0` outputs the same keywords.  I had incorrectly thought that `header0.keys()` includes derived properties such as `nchan`, `payloadsize` and `bps` not returned by printing the header.  Is there a way to include these as well, or should I refer users to the API?

Reply:
It was somewhat of a design choice to have the keywords, ie., anything one can get through `header[key]`, be actual data, and anything one gets through properties be derived.

This was partially why I was wondering whether it wouldn't make sense to expose the derived properties. All it really needs is
```
def properties(self):
    return self._properties
```

Reply:
I think it's sufficient that they're in the API, which will be the case once #84 is resolved.